Update Salesforce troubleshooting section (#2415)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/salesforce-source.md

1. Check **View Event Log Files**
1. Save it

> NOTE: If the error still occurs after following above instructions, please contact Salesforce Support Team as most probably it is a licensing issue that has to be resolved with their help.

Reviewer: ```suggestion
:::note 
If the error still occurs after following the above instructions, contact the Salesforce Support Team. The root cause is likely a licensing issue must be resolved with their help.
:::
```

1. Edit specific Profile which is assigned to the user
1. Go to: **Administrative Permissions** and enable / disable **View Setup and Configuration**. **View Setup and Configuration** should be enabled for access to SetupAuditTrail

> NOTE: If the error still occurs after following above instructions, please contact Salesforce Support Team as most probably it is a licensing issue that has to be resolved with their help.

Reviewer: ```suggestion
:::note 
If the error still occurs after following the above instructions, contact the Salesforce Support Team. The root cause is likely a licensing issue, which requires their help to resolve.
:::
```
